<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransWindow</class>
 <widget class="QWidget" name="TransWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>625</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>625</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Transmusic</string>
  </property>
  <widget class="QToolButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>51</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string>play</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="caView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>255</width>
     <height>255</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="loopBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>194</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Looping</string>
   </property>
   <widget class="QSpinBox" name="steps">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QToolButton" name="loopButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>start loop</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Steps</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>200</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
  <widget class="QToolButton" name="randomButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>224</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>rnd</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>148</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Tempo (bpm)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tempo">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>90</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>191</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Automaton type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="caFamily"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>17</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Automaton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="caSelect"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TransMusic</string>
   </property>
  </widget>
  <widget class="QToolButton" name="recordButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>255</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>record</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="backspase">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
